api/migrationtarget/doc.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2016 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-// This package defines the client side API facade for use by the
+// Package migrationtarget defines the client side API facade for use by the
```

Comment:
Ha! I've just learned that this is the "right" way to do package docstrings in Go.